zhaquirks/nodon/roller_shutter.py

```diff
@@ -0,0 +1,154 @@
+"""NodOn Roller Shutter Relay Switch."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant import UnitOfTime
+from zigpy.quirks.v2.homeassistant.number import NumberDeviceClass
+import zigpy.types as t
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+NODON = "NodOn"
+
+
+class NodOnWindowCovering(WindowCovering, CustomCluster):
+    """NodOn custom WindowCovering cluster."""
+
+    class AttributeDefs(BaseAttributeDefs):
```

Comment:
Is there a reason to not inherit from `WindowCovering.AttributeDefs` here?
If that's done, we could remove all the copied attributes from zigpy below.

Reply:
Ah, the other attributes have the same attribute id. Hmm..

Reply:
Yes, the attributes have same IDs. Happy to inherit and override if there is an easy way.

Reply:
No, it doesn't work of if I just inherit. It was my first approach and it didn't work.

Reply:
Ideally, zigpy should be able to have both defined at the same time, since the original attribute still exists on the cluster, as long as the manufacturer code isn't provided.
Unfortunately, zigpy does not support this yet and it will require some bigger changes.

This approach might be fine for now, since we don't seem to use the attributes that are overridden by the manufacturer ones.